src/main.py

```diff
@@ -0,0 +1,108 @@
+from src import data_handler as dh
+from src import activity_tracker_handler as ath
+from src import consts
+import pandas as pd
+import logging
+import csv
+import sys
+import os
+
+pd.set_option('display.max_rows', 250)
+pd.set_option('display.max_columns', 100)
+
+
+def __get_data_path():
+    args = sys.argv
+    if len(args) < 3 or consts.PATH_CMD_ARG not in args:
+        raise NameError('List of arguments must have required 1 element: path!')
+    try:
+        path = args[args.index(consts.PATH_CMD_ARG) + 1]
+    except ValueError:
+        raise NameError('List of arguments does not have the argument -path')
+    except IndexError:
+        raise NameError('List of arguments does not have a value after the argument -path')
+    return path
+
+
+def __count_containing_substring(list_of_string: list, substring: str):
+    count = 0
+    for i, s in enumerate(list_of_string):
+        if substring in s:
+            count += 1
+    return count
+
+
+def __index_containing_substring(list_of_string: list, substring: str):
+    for i, s in enumerate(list_of_string):
+        if substring in s:
+            return i
+    return -1
+
+
+def __get_real_ati_file_index(path: str, files: list, ati_key: str):
```

Comment:
This function returns the first AT file but it doesn't check the rest of the files. It may be that something went wrong and there are several AT files. Maybe it would be better to check the rest of the files and in case of finding another one AT file throw an exception? We also don't need the __count_containing_substring and __index_containing_substring functions in this case cause we will find the real AT by __get_real_ati_file_index or throw an exception.